### ISSUE TITLE
Restart Loop on 7.0.0

### ISSUE THREAD

**Issue by marciogranzotto** [hassio-addons/addon-node-red#606]
# Problem/Motivation

> After uptating the add-on to 7.0.0, node-red gets in a restart loop and doesn't complete the start up

## Expected behavior

> Should start normally

## Actual behavior

```bash
Welcome to Node-RED
===================
9 Jul 11:49:34 - [info] Node-RED version: v1.1.0
9 Jul 11:49:34 - [info] Node.js  version: v12.17.0
9 Jul 11:49:34 - [info] Linux 5.4.50 x64 LE
9 Jul 11:49:35 - [info] Loading palette nodes
Thu, 09 Jul 2020 14:49:36 GMT node-telegram-bot-api deprecated Automatic enabling of cancellation of promises is deprecated.
In the future, you will have to enable it yourself.
See https://github.com/yagop/node-telegram-bot-api/issues/319. at internal/modules/cjs/loader.js:1138:30
9 Jul 11:49:36 - [info] Dashboard version 2.22.1 started at /endpoint/ui
/bin/sh: /opt/node_modules/node-red-node-pi-gpio/testgpio.py: not found
9 Jul 11:49:36 - [warn] rpi-gpio : Raspberry Pi specific node set inactive
9 Jul 11:49:36 - [info] Settings file  : /etc/node-red/config.js
9 Jul 11:49:36 - [info] Context store  : 'default' [module=memory]
9 Jul 11:49:36 - [info] User directory : /config/node-red/
9 Jul 11:49:36 - [info] Server now running at http://127.0.0.1:46836/
9 Jul 11:49:36 - [info] Active project : home-automation
9 Jul 11:49:36 - [info] Flows file     : /config/node-red/projects/home-automation/flows.json
9 Jul 11:49:36 - [info] Starting flows
9 Jul 11:49:36 - [info] [python3-function:BitBucketConverter] Python function 'BitBucketConverter' running on PID undefined
9 Jul 11:49:36 - [info] [python3-function:BitBucketConverter] Python function 'BitBucketConverter' running on PID undefined
9 Jul 11:49:36 - [info] Started flows
9 Jul 11:49:36 - [red] Uncaught Exception:
9 Jul 11:49:36 - Error: spawn python ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:267:19)
    at onErrorNT (internal/child_process.js:469:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! addon-node-red@ start: `node $NODE_OPTIONS node_modules/node-red/red.js "--settings" "/etc/node-red/config.js"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the addon-node-red@ start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-07-09T14_49_36_978Z-debug.log
[11:49:37] INFO: Starting Node-RED...
> addon-node-red@ start /opt
> node $NODE_OPTIONS node_modules/node-red/red.js "--settings" "/etc/node-red/config.js"
9 Jul 11:49:37 - [info] 
Welcome to Node-RED
```
and repeat

## Steps to reproduce

> Just updated from 6.3.0 to 7.0.0

**Comment by sinclairpaul** [comment]
I would guess you have a palette installed which isn't compatible with the version of node-red running.

Assuming you installed these via Node-Red itself and not in the addon config, you should be able to remove them via the config/node-red folder and the package.json file.

**Comment by marciogranzotto** [comment]
yeah, it seems to work if I remove `node-red-contrib-telegrambot-home`, which is very unfortunate since it is a node that I use a lot.
I noticed that Node-RED released the `1.1.1` version yesterday with several bug fixes. Any chance to update the add-on to it?

**Comment by sinclairpaul** [comment]
There doesn't seem to be anything in the release notes that would be related to your issue.  I would suggest logging the issue at the repo for the problematic flow.
